src/main/ruby/core/truffle/proc_operations.rb

```diff
@@ -25,22 +33,22 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 module Truffle
-  class Mirror
-    class Proc < Mirror
-      def curry(executable, args, arity)
-        args.freeze
+  module ProcOperations
+
+    def self.curry(executable, args, arity)
+      args.freeze
 
-        name = executable.lambda? ? :lambda : :proc
+      name = executable.lambda? ? :lambda : :proc
 
-        Proc.__send__(name) do |*a|
-          all_args = args + a
-          if all_args.size < arity
-            curry executable, all_args, arity
-          else
-            executable[*all_args]
-          end
+      Proc.__send__(name) do |*a|
+        all_args = args + a
+        if all_args.size < arity
+          curry executable, all_args, arity
+        else
+          executable[*all_args]
         end
       end
     end
+    
   end
-end
+end
```

Comment:
Really minor, but I think we should always have a final newline.
Maybe there is a cop for that in RuboCop.
Anyway, doesn't for this PR.

Reply:
I haven't seen this as significant for quite a while. But I have no problem adding them.